### PR TITLE
feat: Add support for FLAC codec

### PR DESCRIPTION
**Pull request by avelad**

### CODE REVIEW

File: streamer/bitrate_configuration.py [hunk 1]
```diff
@@ -154,6 +155,7 @@ def _sortable_properties(self) -> Tuple[float]:
       'opus': '32k',
       'ac3': '96k',
       'eac3': '48k',
+      'flac': '705k',
```
Comment (avelad) [inline streamer/bitrate_configuration.py:158]: I agree to review these values ​​if you consider it.

Comment (joeyparrish) [reply streamer/bitrate_configuration.py:158]: Does FFmpeg even honor them?  FLAC is lossless, so we don't really control the bitrate, do we?

Comment (avelad) [reply streamer/bitrate_configuration.py:158]: It doesn't control it, but we need to add something here to make all our logic work.

Comment (joeyparrish) [reply streamer/bitrate_configuration.py:158]: Understood.  Then these numbers are fine.

**Review by joeyparrish — COMMENTED** [review]
I'd like to merge https://github.com/shaka-project/shaka-streamer/pull/189, retest and merge https://github.com/shaka-project/shaka-streamer/pull/188, then retest and merge this.

After all of those are merged, I want to release 1.0.0.

**Pull request merged**